### ISSUE TITLE
About configuration files

### ISSUE THREAD

**Issue by neversettle-tech** [Sentient07/HeadHunter#14]
Where are the configuration files mentioned in Requirements, such as head_detection.yml, onfig/config_chuman.yaml

**Comment by Johnqczhang** [comment]
I guess `head_detection.yml` refers to `env_super.yml` in this repo, and the author didn't release `config/config_chuman.yaml` so far. However, the three config files in the `config/` folder of this repo correspond to the three supported training schemes mentioned in the [README.md](https://github.com/Sentient07/HeadHunter#training). So, maybe you can follow the details mentioned in these files.

**Comment by neversettle-tech** [comment]
ok,thank you

**Comment by Sentient07** [comment]
We don't report any standalone results on CHuman dataset. We only use it for pre-training and that config can be found here : https://github.com/Sentient07/HeadHunter/blob/master/config/PT/R50_PT.yaml

**Comment by Johnqczhang** [comment]
> We don't report any standalone results on CHuman dataset. We only use it for pre-training and that config can be found here : https://github.com/Sentient07/HeadHunter/blob/master/config/PT/R50_PT.yaml

However, the training hyper-parameters in `R50_PT.yaml` are not consistent with the experimental setup described in your paper. For example, in `R50_PT.yaml`, the base learning rate is set to 0.01 and `max_epoch` is 50, while in your paper, the pre-training on the combination of CrowdHuman and SCUT-HEAD datasets is executed for 20 epochs at a learning rate of 0.001. Thus, in order to reproduce the results shown in the paper, which configuration should I follow? Thanks!

**Comment by Sentient07** [comment]
@Johnqczhang 

To reproduce the results, use our pre-trained model : https://drive.google.com/file/d/1CODnsMrFrOCsE7XL9jBDP42GUz3rbUwH/view?usp=sharing.

To retrain, follow the parameters mentioned in the paper with appropriate Pytorch and Torchvision (0.7.0)
